src/modules/access/appColumns.tsx

```diff
@@ -0,0 +1,74 @@
+import { ColumnDef } from '@tanstack/react-table';
+import { PublishedApporder } from 'iexec/IExecOrderbookModule';
+import CopyButton from '@/components/CopyButton';
+import useUserStore from '@/stores/useUser.store';
+import { getTokenSymbol } from '@/utils/chain.utils';
+import { truncateAddress } from '@/utils/truncateAddress';
+
+export const columns: ColumnDef<PublishedApporder>[] = [
+  {
+    accessorKey: 'order.appprice',
+    header: 'Price',
+    cell: ({ row }) => (
+      <span>
+        {row.original.order.appprice}{' '}
+        {getTokenSymbol(useUserStore.getState().chainId)}
```

Comment:
price in orders is in nRLC, use [formatRLC(nRlcPrice)](https://github.com/iExecBlockchainComputing/iexec-sdk/blob/master/docs/modules/utils.md#formatrlc) to display in RLC unit